Sources/SemanticAnalyzer/SemanticError.swift

```diff
@@ -157,4 +160,8 @@ extension Diagnostic {
   static func contractNotDeclaredInModule() -> Diagnostic {
     return Diagnostic(severity: .warning, sourceLocation: nil, message: "No contract declaration in top level module")
   }
+  
+  static func nonVoidAttemptCall(_ attempt: AttemptExpression) -> Diagnostic {
+    return Diagnostic(severity: .warning, sourceLocation: attempt.sourceLocation, message: "Functions called with try? only return success not the non-Void type yet")
```

Comment:
I find this error message hard to understand. What about `Calling a function returning a non-Void value with try? is not supported yet`?